hyperspy/tests/mva/test_decomposition.py

```diff
@@ -229,3 +253,38 @@ def test_decomposition_supported_return_false(self):
                 algorithm=algorithm,
                 return_info=False,
                 output_dimension=1) is None
+
+
+@pytest.mark.skipif("sys.platform == 'darwin'")
+class TestPlotExplainedVarianceRatio:
+
+    def setup_method(self, method):
+        MY_PATH = os.path.dirname(__file__)
```

Comment:
If you use e.g. ``np.random.seed(1)`` before creating the random dataset then it should always be the same. For example, the following may do:

```python

import numpy as np
import hyperspy.api as hs
np.random.seed(1)
s = hs.signals.Signal1D(np.dot(np.random.random((100, 5)), np.random.exponential(size=(5,100))))

```